client/src/pages/learn/apis-and-microservices/managing-packages-with-npm/index.md

```diff
@@ -5,5 +5,13 @@ superBlock: APIs and Microservices
 ---
 ## Introduction to the Managing Packages with npm Challenges
 
-The Node Package Manager (npm) is a command-line tool used by developers to share and control modules (or packages) of JavaScript code written for use with Node.js.<br><br>When starting a new project, npm generates a <code>package.json</code> file. This file lists the package dependencies for your project. Since npm packages are regularly updated, the <code>package.json</code> file allows you to set specific version numbers for each dependency. This ensures that updates to a package don't break your project.<br><br>npm saves packages in a folder named <code>node\_modules</code>. These packages can be installed in two ways:<br><br><ol><li>globally in a root <code>node\_modules</code> folder, accessible by all projects.</li><li>locally within a project's own <code>node\_modules</code> folder, accessible only to that project.</li></ol><br>Most developers prefer to install packages local to each project to create a separation between the dependencies of different projects.
+The Node Package Manager (npm) is a command-line tool used by developers to share and control modules (or packages) of JavaScript code written for use with Node.js.<br><br>When starting a new project, npm generates a <code>package.json</code> file. This file lists the package dependencies for your project. Since npm packages are regularly updated, the <code>package.json</code> file allows you to set specific version numbers for each dependency. This ensures that updates to a package don't break your project.
+
+npm saves packages in a folder named <code>node\_modules</code>. These packages can be installed in two ways:
+
+
+1. globally in a root <code>node\_modules</code> folder, accessible by all projects.
+2. locally within a project's own <code>node\_modules</code> folder, accessible only to that project.
+
+Most developers prefer to install packages local to each project to create a separation between the dependencies of different projects.
 Working on these challenges will involve you writing your code on Glitch on our starter project. After completing each challenge you can copy your public Glitch url (to the homepage of your app) into the challenge screen to test it! Optionally you may choose to write your project on another platform but it must be publicly visible for our testing.<br>Start this project on Glitch using <a href='https://glitch.com/edit/#!/remix/clone-from-repo?REPO_URL=https://github.com/freeCodeCamp/boilerplate-npm/'>this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-npm/'>this repository</a> on GitHub! If you use Glitch, remember to save the link to your project somewhere safe!
```

Comment:
```suggestion
Working on these challenges will involve you writing your code on Glitch on our starter project. After completing each challenge you can copy your public Glitch url (to the homepage of your app) into the challenge screen to test it! Optionally you may choose to write your project on another platform but it must be publicly visible for our testing.

Start this project on Glitch using <a href='https://glitch.com/edit/#!/remix/clone-from-repo?REPO_URL=https://github.com/freeCodeCamp/boilerplate-npm/'>this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-npm/'>this repository</a> on GitHub! If you use Glitch, remember to save the link to your project somewhere safe!
```

Reply:
Can you add an empty line here so it looks like the rest of the intros. I think this suggestion should do it. Thanks.